src/prefect/client/client.py

```diff
@@ -775,15 +775,19 @@ def set_flow_run_state(
         mutation = {
             "mutation($state: JSON!)": {
                 with_args(
-                    "setFlowRunState",
+                    "setFlowRunStates",
```

Comment:
The way this mutation is written, this construction could be preferable:

```python
mutation = {
    "mutation($input: setFlowRunStatesInput!)": {
        "setFlowRunStates(input: $input)": {"states": {"id"}}
    }
}

self.graphql(
    mutation,
    variables=dict(
        input=dict(states=[dict(flowRunId=flow_run_id, version=version, state=state)])
    ),
)
```

In other words, moving the complexity out of the GraphQL parser and into the `input` object. 

This has no impact other than possible readability, and I only suggest it because you're using a variable anyway (`$state`), so you might as well do everything in a variable.

(code not tested)